source/spirv_stats.cpp

```diff
@@ -108,6 +109,49 @@ class StatsAggregator {
     }
   }
 
+  // Collects OpConstant statistics.
```

Comment:
Are you also interested in OpSpecConstant ?
Those might not be very prevalent in the ecosystem, but we're hoping....